lib/main_micro_alp.pm

```diff
@@ -277,6 +277,23 @@ sub load_slem_on_pc_tests {
     loadtest("publiccloud/ssh_interactive_end", run_args => $args);
 }
 
+sub load_xfstests_tests {
+    loadtest 'microos/disk_boot';
```

Comment:
```suggestion
    load_boot_from_disk_tests;
```

Reply:
ah, there has a standard function. I'll modify it. thx!